dependency-manager-tests/spm/SPMProject/AppDelegate.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AppDelegate.swift
+//  SPMProject
+//
+//  Created by Mert Buran on 26/03/2020.
+//  Copyright © 2020 Datadog. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+import Datadog
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
```

Comment:
Let's add `SPMProject` folder to both `sources.swiftlint.yml` and `tests.swiftlint.yml` and align the template syntax with our convention. Also let's add the build phase to keep notified when updating the project.

```yml
# tools/linter/sources.swiftlint.yml

included:
  - Sources
  - instrumented-tests/Integration/Integration
  - instrumented-tests/Benchmark/Benchmark
  - instrumented-tests/http-server-mock/Sources
  - dependency-manager-tests/spm/SPMProject 👈
```
```yml
# tools/linter/tests.swiftlint.yml

included:
  - Tests
  - instrumented-tests/Integration/IntegrationTests
  - instrumented-tests/Benchmark/BenchmarkTests
  - instrumented-tests/http-server-mock/Tests
  - dependency-manager-tests/spm/SPMProject/Tests 👈
```